mabwiser/base_mab.py

```diff
@@ -191,7 +191,7 @@ def _partition_contexts(self, n_contexts: int):
         n_jobs = self._effective_jobs(n_contexts, self.n_jobs)
 
         # Partition contexts between jobs
-        n_contexts_per_job = np.full(n_jobs, n_contexts // n_jobs, dtype=np.int)
+        n_contexts_per_job = np.full(n_jobs, n_contexts // n_jobs, dtype=int)
```

Comment:
Isn't this in conflict with the added version requirement? And seems like it would cause backward compatibility issues (ie should be bundled into a version increment)

Reply:
This is not in conflict, tests pass. numpy 1.20.0+ would raise a warning, which this change fixes. Additionally, until we are able to bump to sklearn 0.24.0+, numpy 1.20.0+ would raise this deprecation warning inside sklearn itself.

Reply:
It looks like the np.type format is aliases (https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations) which explains this. I didn't know we could directly use them this way!